src/lib.rs

```diff
@@ -2,6 +2,10 @@
 //
 // Use of this source code is governed by a BSD-style license
 // that can be found in the LICENSE file.
+//! Fontsan - a sanitizer for untrusted font files.
```

Comment:
Probably we should either say "FontSan" or "fontsan":

```suggestion
//! fontsan - a sanitizer for untrusted font files.
```